pkg/kubelet/image_manager_test.go

```diff
@@ -85,11 +86,21 @@ func TestDetectImagesInitialDetect(t *testing.T) {
 	fakeRuntime.ImageList = []container.Image{
 		makeImage(0, 1024),
 		makeImage(1, 2048),
+		makeImage(2, 2048),
 	}
 	fakeRuntime.AllPodList = []*containertest.FakePod{
 		{Pod: &container.Pod{
 			Containers: []*container.Container{
-				makeContainer(1),
+				{
+					ID:      container.ContainerID{Type: "test", ID: fmt.Sprintf("container-%d", 1)},
+					ImageID: imageName(1),
+					// The image filed is not set to simulate a no-name image
+				},
+				{
```

Comment:
Just want to make the comparison clear to the reader that one image has no name while the other has one.